docs/developer/architecture/logs.rst

```diff
@@ -0,0 +1,197 @@
+  logs (from all servers) from a unique endpoint, in order to ease Solution
+  instance Error investigation.
+
+- As a Platform Administrator, I want to push all containers logs to an
+  external log centralisation system, In Order to archive it or aggregate it
+  with other application logs.
+
+
+(some other US extracted from Loki design doc)
+
+- After receiving an alert on my service and drilling into the query associated
+  with said alert, I want to quickly see the logs associated with the jobs
+  which produced those timeseries at the time of the alert.
+
+- After a pod or node disappears, I want to be able to retrieve logs from just
+  before it died, so I can diagnose why it died.
+
+- After discovering an ongoing issue with my service, I want to extract a
+  metric from some logs and combine it with my existing time series data.
+
+- I have a legacy job which does not expose metrics about errors - it only logs
+  them.  I want to build an alert based on the rate of occurrences of errors in
+  the log.
+
+
+Deployment & Configuration
+--------------------------
+
+The log centralisation service is scheduled on infra nodes.
+A platform Administrator can operate the service as follow:
+
+- add persistent storage
+- configure max retention period
+- adjust the number of replicas
+- push logs to an external log centralisation service
+- configure alerting related to log centralisation service
+
+Those operations are accessible through kubectl, kubectl metalk8s plugin or
+metalk8scli.
+
+When installing or upgrading MetalK8s, the log centralisation service is
+automatically scheduled (as soon as a persistent volume is provisioned) on one
+infra node.
+
+During future MetalK8s upgrades, the service stays available (when replicated).
+
+Monitoring
+----------
+
+- An alert rule is fired when the log centralisation service is not healthy
+- The log centralisation service is not healthy when log storage is getting
+  full or when service is not able to ingest logs at the right pace.
+- IOPS, bandwidth, latency, capacity usage KPIs are available in Prometheus
+
+UI
+--
+
+Logs can be seen in Grafana.
+Log centralisation monitoring information are displayed in the MetalK8s UI
+overview page.
+A Grafana dashboard gathering health/performance KPIs, as well as alerts for
+log centralisation service is available when deploying/upgrading MetalK8s.
+
+Documentation
+-------------
+The sizing section in Introduction page is updated to include log
+centralisation service impact. The sizing rule takes in account the retention
+period, workloads expected log rate and workload pre defined indices. This rule
+is to be known by solution developers to properly size the service based on
+the workload properties.
+
+The Post Installation page is updated to indicate that persistent storage is
+needed for log centralisation service.
+
+A new section should be added under Cluster and Configuration services explains
```

Comment:
IMHO, we should remove the CSC section of the Operations guide, it doesn't mean anything for a user. Instead, keep focused pages, each dedicated to a single service.

Reply:
done